deep_quoridor/src/quoridor.py

```diff
@@ -378,6 +393,22 @@ def get_valid_wall_actions(self, player: Optional[Player] = None) -> list[WallAc
     def get_valid_actions(self, player: Optional[Player] = None) -> Sequence[Action]:
         return self.get_valid_move_actions(player) + self.get_valid_wall_actions(player)
 
+    def rotate_action(self, action):
+        """
+        Get the equivalent action for a rotated board.
+        """
+        if isinstance(action, MoveAction):
+            return MoveAction(
+                (self.board.board_size - 1 - action.destination[0], self.board.board_size - 1 - action.destination[1])
+            )
+        elif isinstance(action, WallAction):
+            return WallAction(
+                (self.board.board_size - 2 - action.position[0], self.board.board_size - 2 - action.position[1]),
+                action.orientation,
+            )
```

Comment:
Are you sure about this one? I'd think that rotating a horizontal wall would be different than rotating a vertical wall, because the walls are 2 in length and we use the top left corner, so when you rotate you need to account for the wall going "in the opposite direction".  E.g. a horizontal wall in the top left corner of the board would move to the bottom right, but you need to move it one to the left or it will overflow.
I think having tests for this would help